nozzle/api_client.go

```diff
@@ -96,55 +110,40 @@ func (api *APIClient) listApps() map[string]*AppInfo {
 	return appsInfo
 }
 
+func (api *APIClient) GetUncached(key string) (*AppInfo, error) {
+	app, err := api.client.AppByGuid(key)
+	if err != nil {
+		return nil, err
+	}
+	return newAppInfo(app), nil
+}
+
 // GetApp return cached AppInfo for a guid
 func (api *APIClient) GetApp(guid string) (*AppInfo, error) {
-	size := metrics.GetOrRegisterGauge("cache.size", nil)
-	errors := metrics.GetOrRegisterCounter("cache.errors", nil)
-	miss := metrics.GetOrRegisterCounter("cache.miss", nil)
-	size.Update(int64(api.appCache.ItemCount()))
+	//size := metrics.GetOrRegisterGauge("cache.size", nil)
+	// size.Update(int64(api.appCache.ItemCount()))
 
 	appInfo, found := api.appCache.Get(guid)
 	if found {
 		return appInfo.(*AppInfo), nil
 	}
 
+	miss := metrics.GetOrRegisterCounter("cache.miss", nil)
```

Comment:
@laullon we should use the `go-metrics-wavefront` reporter and include the pcf environment tag on all of these metrics.